docs/index.html

```diff
@@ -224,16 +182,22 @@ <h3>ylem</h3>
 								    };
 								  }
 
+								  /* No more callbacks */
+
 								  render() {
 								    return (
 								      &lt;div&gt;
+								      	/* Pass state to children */
 								        &lt;NextPrev paginate={this.state} /&gt;
 								        &lt;Grid paginate={this.state} /&gt;
 								      &lt;/div&gt;
 								    );
 								  }
 								}
 								</code></pre>
+								<div style="padding: 195px 0; color: #ccc; text-align: center">
```

Comment:
for zooming and future-forward purposes, probably better to use `em` here instead of `px`

Reply:
I'm an `em`-only type of guy, but if a project isn't using it then I don't either.